src/detection/card/use-items.js

```diff
@@ -0,0 +1,4 @@
+import { useSelector } from 'react-redux'
+import { get } from 'lodash'
+
+export const useItems = () => useSelector(({ detection }) => get(detection, 'get.data', []))
```

Comment:
Well, it's only one item, so why name this hook `useItems`?

Also, since we're already here, let's change this function's default value from an empty array to something else (an empty object maybe?)

Reply:
done